### ISSUE TITLE
Error on import: `bad ws edit: []`

### ISSUE THREAD

**Issue by 95th** [rust-analyzer/rust-analyzer#5913]
![image](https://user-images.githubusercontent.com/22871609/91636460-55f10a00-ea1e-11ea-869d-41305064144e.png)

Extension version: v0.2.289

```
$ rust-analyzer-linux --version
rust-analyzer e65d48d
```

**Comment by 95th** [comment]
MCVE:

```rust
use token::TokenKind::*;

fn token(kind: TokenKind) {  // <-- try control + '.' to auto import `TokenKind`
    todo!()
}

mod token {
    pub enum TokenKind {}
}
```

**Comment by kjeremy** [comment]
It works if I remove
```rust
use token::TokenKind::*;
```

**Comment by Veykril** [comment]
This can be closed as it was fixed in #5935.
